commons-rest/src/test/java/com/mule/connectors/commons/rest/builder/handler/RequestTest.java

```diff
@@ -49,4 +52,32 @@ public void testAddPathParam() {
         this.request.addPathParam(DEFAULT_KEY, DEFAULT_VALUE);
         verify(mock);
     }
+
+    /**
+     * This test is made so that all the getter methods on Request are used. These methods are for external use and as such this is the only way to test them.
+     */
+    @Test
+    public void testEnsureCoverage() {
```

Comment:
I would have called this method something like "testGettersAndSetters", but it's up to you.